AzureAttestationExtension/manifest.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version='1.0' encoding='utf-8' ?>
+<ExtensionImage xmlns="http://schemas.microsoft.com/windowsazure">
+  <ProviderNameSpace>Microsoft.Azure.TestAttestation</ProviderNameSpace>
```

Comment:
why the provider namespace is named as TestAttestation, is this manifest file for test extension? I am assuming one for the actual extension as well?

Reply:
discussed offline